src/Json.Schema.ToDotNet.UnitTests/TestData/Integer.schema.json

```diff
@@ -0,0 +1,74 @@
+﻿{
+  "type": "object",
+  "properties": {
+    "integerProperty_default": {
+      "type": "integer"
+    },
+    "integerProperty_min_0_max_na": {
+      "type": "integer",
+      "minimum": 0
+    },
+    "integerProperty_min_minus1_max_na": {
+      "type": "integer",
+      "minimum": -1
+    },
+    "integerProperty_min_1_max_na": {
+      "type": "integer",
+      "minimum": 1
+    },
+    "integerProperty_min_na_max_0": {
+      "type": "integer",
+      "maximum": 0
+    },
+    "integerProperty_min_0_max_int64times10": {
```

Comment:
>int64times10 [](http://example.com/codeflow?start=31&length=12)

for int32 it is add1, but int64 it is times10,
the reason is min and max is double type,
double is less accurate when number is big.

min and max can not be changed to BigInterger because 
min and max definition is not integer only.